extension.json

```diff
@@ -12,15 +12,19 @@
 	"license-name": "GPL-3.0-or-later",
 	"type": "specialpage",
 	"requires": {
-		"MediaWiki": ">= 1.40.0"
+		"MediaWiki": ">= 1.40.0",
```

Comment:
Bump to 1.42 and branch REL1_40 and REL1_41 first. Branches will only ne kept in between LTS releases though so REL1_39-REL1_42 will be deleted when REL1_39 goes EOL